swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -175,89 +164,91 @@ let default_translations = {
         share: "Share",
         share_id: "Share ID",
         share_id_tooltip:
-          "<strong>When another project wants to share a folder " +
-          "with your project</strong><br/>Select this button and " +
-          "send the Share ID (now copied in your cache) " +
-          "to the project's member.<br/><br/> " +
-          "<strong>When you want to share a folder with " +
-          "another project</strong><br/>Ask the Share ID from " +
-          "another project's member.",
+          "With this action, you can copy the Share ID: " +
+          "a unique 32-digit code associated with your <b>currently " +
+          "selected</b> CSC project. Provide the Share ID to members " +
```

Comment:
This should not say "CSC".

Reply:
That's how Aino and Francesca worded it. What would be the issue here?

Reply:
Branding strings go to a different repository

Reply:
Okay, isn't that something you were working on?

Reply:
Yes, that is what I had done, and I'm not going to do it again ^^.

Reply:
I see :D I think it makes sense to get all the UX language up to par with Aino first, and then think of neutral language and move the branding. Since the language has had quite a few changes, and there's probably still some coming in the next issues, there would be a lot of back-and-forth. I don't mind dealing with the language stuff, but we'll need your input, if you're willing

Reply:
I think my point was that we're not adding branding language to this repo. But you could, for e.g. use this branch as a base branch for the other changes, until the work with the strings is done. Just mark the PR with `WIP: ` or draft, then please.

Reply:
Sounds good